models/doctype/Event/Event.js

```diff
@@ -35,7 +35,12 @@ module.exports = {
       fieldtype: 'Table',
       childtype: 'EventSchedule',
       label: 'Schedule'
-    }
+    },
+    {
+      fieldname: "eventposter",
```

Comment:
is this required?

Reply:
nope, was just testing the file, forgot to remove.